### ISSUE TITLE
Update assignsiteid.py once feature_phenology_biomass_scaling is merged into streamline_output

### ISSUE THREAD

**Issue by kpalmqui** [DrylandEcology/rSFSTEP2#237]
`assignsiteid.py `is now object-oriented. As such, the changes in assignsiteid.py that were implemented within the `feature_phenology_biomass_scaling` branch/milestone can be simplified once `feature_phenology_biomass_scaling ` is folded into `streamline_output `and before `streamline_output i`s merged into `master`. 

See below for a screenshot of this conversation and also see attached for the assignsiteid.py file as it stands in phenology_scaling.

[assignsiteid.txt](https://github.com/DrylandEcology/rSFSTEP2/files/4602219/assignsiteid.txt)

<img width="720" alt="Screen Shot 2020-05-07 at 8 45 11 AM" src="https://user-images.githubusercontent.com/7769797/81457627-1f803e00-9154-11ea-9c5f-ccc703331d6c.png">

**Comment by kpalmqui** [comment]
Resolved by db85c47. 

Tested on Teton and this works as expected.
